themes/horizon/scss/welcome-banner.scss

```
@@ -22,6 +18,10 @@
     margin-block: 0;
     padding-block: var(--space-4) var(--space-8);
 
+    .--with-bg-img & {
+      padding-block: clamp(var(--space-4), 10vh, var(--space-11));
```

```suggestion
      padding-block: clamp(var(--space-4), 10dvh, var(--space-11));
```

wdyt, if it makes sense to use dynamic `dvh` - makes sizing adaptive in mobile (on/off keyboard, URL address bar, etc.)

---

you right, dropping my suggestion then